server/databases/postgres/config.js

```diff
@@ -2,7 +2,7 @@ const fs = require("fs");
 const path = require("path");
 const Sequelize = require("sequelize");
 
-const sequelize = new Sequelize("workouts", "postgres", "", {
+const sequelize = new Sequelize("workouts", "postgres", "null", {
```

Comment:
Make sure you change this back to an empty string for your set up. For some reason, Postgres won't let me set my password to an empty string.